src/ChannelCallback.cpp

```diff
@@ -0,0 +1,21 @@
+#include "Globals.h"  // NOTE: MSVC stupidness requires this to be the same across all modules
```

Comment:
Usually we put a header comment into the file saying what it is and what it does, high-level. So:
```cpp
// ChannelCallback.cpp

// Implements the cChannelCallback class representing the callbacks used for plugin channel management
```